tests/integration/targets/prepare_soap/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Install dependencies
+  ansible.builtin.pip:
+    name:
+      - requests
```

Comment:
I think it's better to have it in one place. Either here or in `tests/integration/requirements.txt` as you've created.

Reply:
It didn't work to point to the requirements.txt file :
```
---
- name: Install dependencies
  ansible.builtin.pip:
    rrequirements: requirements.txt
```